.github/workflows/build_images.yml

```diff
@@ -162,6 +162,8 @@ jobs:
       - name: Run vulnerability scan
         id: vuln_scan
         uses: aquasecurity/trivy-action@0.14.0
+        env:
+          TRIVY_DB_REPOSITORY: "public.ecr.aws/aquasecurity/trivy-db:2"
```

Comment:
the default is `ghcr.io/aquasecurity/trivy-db`. this gets it from here https://gallery.ecr.aws/aquasecurity/trivy-db instead which may be more reliable

Reply:
Hmm, seems this failed as well.  @yakloinsteak is this a different error?

Reply:
Try upgrading the trivy action version to 0.28.0. I'll take a look if that fails.

Reply:
Updated

Reply:
@yakloinsteak still failing.